app/src/main/contrib/components/Responsibilities/BurdenEstimates/UploadBurdenEstimatesPage.tsx

```diff
@@ -1,29 +1,37 @@
 import * as React from "react";
-import { touchstoneActions } from "../../../actions/TouchstoneActions";
-import { responsibilityActions } from "../../../actions/ResponsibilityActions";
-import { responsibilityStore } from "../../../stores/ResponsibilityStore";
-import { modellingGroupActions } from "../../../../shared/actions/ModellingGroupActions";
-import { PageWithHeaderAndNav } from "../../PageWithHeader/PageWithHeaderAndNav";
-import { doNothing } from "../../../../shared/Helpers";
-import { DownloadDataTitle } from "../DownloadDataTitle";
-import { UploadBurdenEstimatesContent } from "./UploadBurdenEstimatesContent";
-import { estimateTokenActions } from "../../../actions/EstimateActions";
+import {touchstoneActions} from "../../../actions/TouchstoneActions";
+import {responsibilityActions} from "../../../actions/ResponsibilityActions";
+import {responsibilityStore} from "../../../stores/ResponsibilityStore";
+import {modellingGroupActions} from "../../../../shared/actions/ModellingGroupActions";
+import {PageWithHeaderAndNav} from "../../PageWithHeader/PageWithHeaderAndNav";
+import {doNothing} from "../../../../shared/Helpers";
+import {DownloadDataTitle} from "../DownloadDataTitle";
+import {UploadBurdenEstimatesContent} from "./UploadBurdenEstimatesContent";
+import {estimateTokenActions} from "../../../actions/EstimateActions";
+import {
+    processEncodedResultAndNotifyOnErrors
+} from "../../../../shared/sources/Source";
+import {queryStringAsObject} from "../../../../shared/Helpers";
 
-interface LocationProps {
+export interface UploadEstimatesProps {
     groupId: string;
     touchstoneId: string;
     scenarioId: string;
 }
 
-export class UploadBurdenEstimatesPage extends PageWithHeaderAndNav<LocationProps> {
+export class UploadBurdenEstimatesPage extends PageWithHeaderAndNav<UploadEstimatesProps> {
     componentDidMount() {
         setTimeout(() => {
+
+            processEncodedResultAndNotifyOnErrors<string>(queryStringAsObject());
```

Comment:
Later, I think we should invoke this in `PageWithHeader`, so that all pages get this behaviour. For now though, I'm happy to keep this change less far reaching. Also, it would be better to leave changes to `PageWithHeader` until VIMC-397 is finished.